### PR TITLE
docs: add piece store underlying storage docs

### PR DESCRIPTION
**Pull request by sysvm**

### CODE REVIEW

File: docs/guide/storage-provider/run-book/piece-store.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
+---
+title: How to deploy Piece Store
+order: 5
+---
+
+Greenfield SP is a storage infrastructure for Greenfield decentralized storgage platform. Greenfield SP use `PieceStore` to store users payload data.
```
Comment (RenRick) [inline docs/guide/storage-provider/run-book/piece-store.md:6]:
```suggestion
Greenfield SP is a storage infrastructure for Greenfield decentralized storgage platform. Greenfield SP uses `PieceStore` to store users' payload data.
```

File: docs/guide/storage-provider/run-book/piece-store.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+# PieceStore Config
+
+When creating a PieceStore, there are the following options to deploy underlying storage:
```
Comment (RenRick) [inline docs/guide/storage-provider/run-book/piece-store.md:10]:
```suggestion
When creating a PieceStore, there are the following options to configure underlying storage:
```

File: docs/guide/storage-provider/run-book/piece-store.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+[PieceStore]
+Shards = 5
+
+[PieceStore.Store]
+Storage = 'minio'
+BucketURL = 'http://10.180.42.161:9000/mybucket%d'
+IAMType = 'AKSK'
+```
+
+After SP is initializing through the above command, PieceStore will create 5 buckets named `mybucket0`, `mybucket1`, `mybucket2`, `mybucket3` and `mybucket4`.
```
Comment (RenRick) [inline docs/guide/storage-provider/run-book/piece-store.md:55]:
```suggestion
After SP initialized through the above command, PieceStore will create 5 buckets named `mybucket0`, `mybucket1`, `mybucket2`, `mybucket3` and `mybucket4`.
```

File: docs/guide/storage-provider/run-book/piece-store.md (hunk 1 above)
Comment (sysvm) [reply docs/guide/storage-provider/run-book/piece-store.md:6]: fixed

File: docs/guide/storage-provider/run-book/piece-store.md (hunk 2 above)
Comment (sysvm) [reply docs/guide/storage-provider/run-book/piece-store.md:10]: fixed

**Review by maxnodereal — APPROVED** [review]

**Pull request merged**